src/main/scala/uk/gov/nationalarchives/draftmetadatavalidator/Lambda.scala

```diff
@@ -185,6 +187,29 @@ class Lambda {
     } else { IO.unit }
   }
 
+  private def validateAdditionalHeaders(validationParameters: Lambda.ValidationParameters): IO[Unit] = {
+    def additionalError(header: String): Error = {
+      val validationKey = convertToValidationKey(validationParameters.clientAlternateKey, header)
+      val errorKey = if (validationKey.isEmpty) header else validationKey
+      val additionalFileError = FileError.ADDITIONAL_HEADER.toString
+      Error(additionalFileError, header, "additional", s"$additionalFileError.$errorKey.additional")
```

Comment:
I think you don't need to construct `message` as there won't be any error report. We are unnnecessry calling the `convertToValidationKey` method. You can simply generate the error:
```
    def additionalError(header: String): Error = Error(FileError.ADDITIONAL_HEADER.toString, header, "additional", "")

```